src/libgap-api.c

```diff
@@ -25,6 +25,7 @@
 #include "gasman_intern.h"
 #endif
 #include "gvars.h"
+#include "funcs.h"
```

Comment:
We sort includes alphabetically (resp. `clang-format` does for us). Can you please adjust that, and rebase this PR? Then I'll merge and backport it.

Alternatively, I can do this for you so you don't have extra work, just let me know.

Thanks!

Reply:
I'll take care of it.  I actually did notice that they were sorted but for some reason my brain glitched and thought f was after g :)

Reply:
Fixed.